src/lib.rs

```diff
@@ -1,4 +1,4 @@
-//! [`ExpiringMap`] is a wrapper around [`AHashMap`] that allows the specification
+//! [`ExpiringMap`] is a wrapper around [`HashMap`] that allows the specification
```

Comment:
Sorry to nit these- Yes, they are wrong. Do you mind fixing them in another PR? I could also do it if you don't want to.

Reply:
Done